VTEX - Shopper Data API.json

```diff
@@ -0,0 +1,718 @@
+                                                "isCardToken": {
+                                                    "type": "boolean",
+                                                    "description": "Indicates whether the card data is tokenized."
+                                                },
+                                                "availableAddresses": {
+                                                    "type": "array",
+                                                    "description": "Array of address IDs the card is associated with.",
+                                                    "items": {
+                                                        "type": "string",
+                                                        "description": "Address identifier."
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            },
+                            "additionalProperties": {
+                                "type": "boolean",
+                                "description": "When `true`, allows clients to send and persist custom fields beyond the defined schema. This enables storing extra fields like `phone2`, `marketingOptIn`, or any shopper-specific data."
+                            },
+                            "required": {
+                                "type": "array",
+                                "items": {
+                                    "type": "string",
+                                    "description": "Name of a required property field."
+                                },
+                                "description": "Required properties fields."
+                            },
+                            "v-indexed": {
+                                "type": "array",
+                                "items": {
+                                    "type": "string",
+                                    "description": "Name of an indexed property field."
+                                },
+                                "description": "Indexed properties fields."
+                            }
+                        },
+                        "required": ["title", "type", "properties"]
+                    }
+                },
+                "required": ["name", "schema"]
+            },
+            "ShopperDocumentRequest": {
+                "type": "object",
+                "description": "Shopper document creation request. Supports additional properties beyond the defined schema.",
+                "required": [
+                    "userId",
+                    "firstName",
+                    "lastName",
+                    "document",
+                    "documentType"
+                ],
+                "properties": {
```

Comment:
Does it make sense to add a more explicit mention to additional properties in the schema? Maybe a generic field named something like `[additional property]`, explaining this mechanic in the description, mentioning the examples you used in the request example (`phone2` and `marketingOptIn`).